docs/troubleshooting/25_social-sign-in.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: troubleshooting-social-sign-in
+title: Social Sign-in Troubleshooting
+sidebar_label: Social Sign-in Troubleshooting
+slug: troubleshooting-social-sign-in
+---
+
+### `redirect_uri` mismatch
+
+Ensure that the `redirect_uri` query parameter in the URL matches the authorized redirect URI added to the social provider.
+
+```
+https://<provider>/o/oauth2/v2/auth?client_id=(...)&
+redirect_uri=https://{project.slug}.projects.oryapis.com/self-service/methods/oidc/callback/<provider>&
+response_type=code&scope=email+profile+openid
+```
+
+:::info
+
+This is also a known problem with projects created before September 2022 that use a custom domain. Please see
+[Social sign-in `redirect_uri_mismatch` error](./oidc-redirect-url-cname-OTA-092022-01) for further information.
+
+:::
+
+### Domain `oryapis.com` requires verification
```

Comment:
I don't think this heading accurately represents the issue you describe here. Something like this will be more fitting(?): 
```suggestion
## 'oryapis.com' doesn't work as authorized redirect URI
```